packages/familie-header/src/Header.less

```diff
@@ -0,0 +1,134 @@
+.rad {
+  display: flex;
+  align-items: center;
+}
+
+.gap {
+  > *:not(:last-child) {
+    margin-right: 1rem;
+  }
+}
+
+.kolonne {
+  display: flex;
+  flex-direction: column;
+}
+
+.headerTekst() {
+  font-family: 'Source Sans Pro', Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  text-align: left;
+  color: #fff;
+  padding: 0;
+  margin: 0;
+}
+
+.tekstNormal {
+  .headerTekst();
+  font-size: 15px;
+}
+
+.tekstLiten {
+  .headerTekst();
+  font-size: 12px;
+}
+
+.button() {
+  border: none;
+  background: none;
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  cursor: pointer;
+
+  &:hover {
+    background: #59514b;
+  }
+
+  &:active {
+    background: none;
+  }
+
+  &:focus, &:active {
+    outline: none;
+    box-shadow: inset 0 0 0 3px #ffa733;
+  }
+}
+
+.systemknapp {
+  width: 50px;
+  height: 50px;
```

Comment:
Dette er kanskje et dumt spørsmål. Men hvorfor er disse definert med piksler og ikke rem? Betyr ikke dette at man fikserer størrelsen på knappen uavhengig av skjermstørrelse?

Reply:
It is copied from helse-frontend-moduler and I think you are right. This will be fixed.👍

Reply:
The fight between rem and px :) Actually the difference in common browsers today enables us to use both rem and px. However, we use rem for everything else - so I suggest we keep using it. For minor details where we use 1px I still think we should use px.